cppcache/integration/framework/TestConfig.cpp.in

```diff
@@ -22,6 +22,8 @@ const char *getFrameworkString(FrameworkVariable name) {
     case FrameworkVariable::JavaObjectJarPath: return "@JAVAOBJECT_JAR_PATH@";
     case FrameworkVariable::GfShExecutable: return "@Geode_gfsh_EXECUTABLE@";
     case FrameworkVariable::TestCacheXmlDir: return "@CMAKE_CURRENT_SOURCE_DIR@/../../integration-test/resources";
+    case FrameworkVariable::TestClientSslKeysDir: return "@CMAKE_CURRENT_SOURCE_DIR@/../../../ssl_keys/client_keys";
+    case FrameworkVariable::TestServerSslKeysDir: return "@CMAKE_CURRENT_SOURCE_DIR@/../../../ssl_keys/server_keys";
```

Comment:
Why is this a giant case statement. Why not make these all global `const` values in a header. Would be nice to have this be a compile time failure if you are using something not defined than runtime.

Reply:
This is a really nice improvement - tracked as GEODE-8820.

Reply:
https://issues.apache.org/jira/browse/GEODE-8220